regulations/static/regulations/js/source/views/sidebar/sidebar-view.js

```diff
@@ -34,10 +34,16 @@ var SidebarView = Backbone.View.extend({
         this.listenTo(this.externalEvents, 'breakaway:open', this.hideChildren);
 
         this.childViews = {};
+        this.viewlessChildren = {};  /* Children that don't have unique views */
```

Comment:
Minor, but it looks like `viewlessChildren` is expected to be an array elsewhere, so it might be clearer to initialize to an empty array here. I'm also wondering if we could store this as a jQuery variable instead of a list:

```
this.$viewlessChildren = this.$el.find('.regs-meta');
```

and later...

```
this.$viewlessChildren.remove();
```

Finally, why do we need to track this variable at all--would the return value of `this.$el.find('.regs-meta')` change between calling `openRegFolders` and `closeChildren`? I assume it could, since we're adding a variable to track it, but I'm not sure where that change would happen.

#hairsplitting